cmd/contour/serve.go

```diff
@@ -136,6 +136,7 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 	serve.Flag("contour-cafile", "CA bundle file name for serving gRPC with TLS.").Envar("CONTOUR_CAFILE").StringVar(&ctx.caFile)
 	serve.Flag("contour-cert-file", "Contour certificate file name for serving gRPC over TLS.").PlaceHolder("/path/to/file").Envar("CONTOUR_CERT_FILE").StringVar(&ctx.contourCert)
 	serve.Flag("contour-key-file", "Contour key file name for serving gRPC over TLS.").PlaceHolder("/path/to/file").Envar("CONTOUR_KEY_FILE").StringVar(&ctx.contourKey)
+	serve.Flag("contour-log-file", "Optional Logging file path for the controller.").PlaceHolder("/path/to/file").Envar("CONTOUR_LOG_FILE").StringVar(&ctx.Config.ContourLogFile)
```

Comment:
We're looking to deprecate these config flags, so I'd remove this and just utilize the config file.